lang/rust/files/patch-src-librustc-llvm-lib.diff

```diff
@@ -0,0 +1,8 @@
+--- src/librustc_llvm/lib.rs.orig	2018-01-15 17:44:30.000000000 +0400
++++ src/librustc_llvm/lib.rs	2018-01-16 02:36:38.000000000 +0400
+@@ -429,3 +429,5 @@
+         }
+     }
+ }
++
++#[link(name = "ffi")] extern {}
```

Comment:
What does this patch do? Can you add some comments in this file or before `patchfiles` in the Portfile?

Reply:
I believe I got it from rust-lang/rust#34486.